build.sh

```diff
@@ -10,7 +10,7 @@ $ESBUILD --format=cjs --target=es2021 --platform=node lib/web.ts --outfile=dist/
 
 # Extension
 $ESBUILD addon/background.ts --outfile=dist/addon-mv3/background.bundle.js
-$ESBUILD addon/content.ts --outfile=dist/addon-mv3/content.bundle.js
+$ESBUILD addon/content.ts --minify --outfile=dist/addon-mv3/content.bundle.js
```

Comment:
Is there a good reason to minify this for the dev extension? This will come at a cost of making debugging internals more difficult, unless we also generate sourcemaps.

Reply:
Not really, I agree we can rollback. I only did this for performance testing.